src/Appointments.Console/Appointment Management System/Repositories/CustomerRepository.cs

```diff
@@ -10,102 +10,27 @@ namespace AppointmentManagementSystem.Repositories
 {
     public class CustomerRepository: IManagementRepository<Customer>
     {
-         public void Create(List<Customer> customers, List<Customer>? item = null)
-        {
-            Console.WriteLine("Create Customer");
-            Console.WriteLine("---------------");
-
-            Console.Write("Enter Name:");
-            string name = Console.ReadLine()??"";
-            string email;
-            while (true)
-            {
-                Console.Write("Enter Email: ");
-                email = Console.ReadLine()??"";
-                if (Utilities.IsValidEmail(email))
-                {
-                    break;
-                }
-                Console.WriteLine("Invalid email format. Please try again.");
-            }
-            string phoneNumber;
-            while (true)
-            {
-                Console.Write("Enter Phone Number: ");
-                phoneNumber = Console.ReadLine()??"";
-                if (Utilities.IsValidPhoneNumber(phoneNumber))
-                {
-                    break;
-                }
-                Console.WriteLine("Invalid phone number. Please try again.");
-            }
+        private readonly List<Customer> _customers =[];
 
-            customers.Add(new Customer(name, email, phoneNumber));
-            Console.WriteLine("Customer created successfully.");
+        public void Add(Customer customer)
+        {
+            _customers.Add(customer);
         }
 
-        public void Read(List<Customer> customers)
+        public List<Customer> Get()
```

Comment:
Repository exposing it's internal list make it possible to mutate the underlying peristency without the use of the repository (violates tha persistency isolation principle)